Participants.md

```diff
@@ -24,3 +22,6 @@ We will also update the progress and all results/implemenations will be related
 * Eunsoo Park: An automated summarization
 * Xihan Li: One-shot learning using NTM
 * Kyushik Min: Deep Reinforcement Learning Based Autonomous Vehicle Control 
+
+* Yandong Liu: Using Tensorflow to build self-driving cars! (Strike-out)
```

Comment:
Instead of writing (Strike-out), you should wrap the row with`~~`  
```text
* ~~Yandong Liu: Using Tensorflow to build self-driving cars!~~
```
so it looks like this
* ~~Yandong Liu: Using Tensorflow to build self-driving cars!~~